src/styles/styles.js

```diff
@@ -1580,6 +1580,33 @@ const styles = {
         height: 80,
     },
 
+    chatAvatar: {
+        height: 52,
+        width: 52,
+        borderRadius: 100,
+        borderColor: colors.white,
+        borderWidth: 3,
+        marginLeft: -15,
+    },
+
+    chatOverLay: {
+        position: 'absolute',
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        backgroundColor: colors.dark,
+        opacity: 0.5,
```

Comment:
so just the background color should have opacity. I think this is causing the text (`+X`) to be slightly transparent too, which we don't want.

Reply:
@shawnborton i didn't understand what you are trying to say